modules/ROOT/pages/type-definitions/directives/cypher.adoc

```diff
@@ -154,6 +154,26 @@ type Query {
 
 The downside of the latter approach is that you need to adjust the return object as you change your object type definition.
 
+== Input arguments
+The `@cypher` statement can access the query parameters by prepending `$` to the parameter name. For example:
+
+[source, graphql, indent=0]
+----
+type Query {
+    name(value: String): String @cypher(statement: "RETURN $value AS res", columnName: "res")
+}
+----
+
+The following GraphQL query returns the parameter `value`:
+
+[source, graphql, indent=0]
+----
+query {
```

Comment:
another point of confusion on my part: how does this query relate to the query above with the cypher directive in it?

Reply:
In this query we are passing a parameter called `value` that is caught by the `@cypher` directive (see the `$value`)